src/Fuzz.elm

```diff
@@ -640,71 +488,52 @@ and so this function is best used as a helper when creating more complicated fuz
 -}
 constant : a -> Fuzzer a
 constant x =
-    Internal.Fuzzer
-        (\noShrink ->
-            if noShrink then
-                Gen (Random.constant x)
-            else
-                Shrink (Random.constant (RoseTree.singleton x))
-        )
+    Ok <| Random.constant (RoseTree.singleton x)
 
 
 {-| Map a function over a fuzzer. This applies to both the generated and the shrunken values.
 -}
 map : (a -> b) -> Fuzzer a -> Fuzzer b
-map transform (Internal.Fuzzer baseFuzzer) =
-    Internal.Fuzzer
-        (\noShrink ->
-            case baseFuzzer noShrink of
-                Gen genVal ->
-                    Gen <| Random.map transform genVal
-
-                Shrink genTree ->
-                    Shrink <| Random.map (RoseTree.map transform) genTree
-
-                InvalidFuzzer reason ->
-                    InvalidFuzzer reason
-        )
-
-
-{-| Map over two fuzzers.
--}
-map2 : (a -> b -> c) -> Fuzzer a -> Fuzzer b -> Fuzzer c
-map2 transform fuzzA fuzzB =
-    map (\( a, b ) -> transform a b) (tuple ( fuzzA, fuzzB ))
+map =
+    Internal.map
```

Comment:
To be honest I'm not sure if at this point there's any value to the `Internal` module, considering it's no longer being used to hide implementation details. How do you feel about moving everything into `src/Fuzz.elm`?